README.md

```diff
@@ -107,20 +117,24 @@ vaults account create [FILENAME]
 
 A new private key will be written to `FILENAME`.
 
-The name of a vault contains a `namespace` (e.g. `my_company`) and the name of an existing database relation (e.g. `my_table`), separated by a period (`.`). Use `vaults create` to create a new vault. See `vaults create --help` for more info.
+The name of a vault contains a `namespace` (e.g. `my_company`) and an identifier (e.g., `my_data`), separated by a period (`.`). Use `vaults create` to create a new vault. See `vaults create --help` for more info.
```

Comment:
idk if the `namespace.identifier` is the right way to define it, but i figured the "relation" nomenclature should be dropped.